proto/axelar/axelarnet/v1beta1/events.proto

```diff
@@ -80,6 +80,7 @@ message FeePaid {
   bytes recipient = 2 [ (gogoproto.casttype) =
                             "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
   cosmos.base.v1beta1.Coin fee = 3 [ (gogoproto.nullable) = false ];
+  string refund_recipient = 4;
```

Comment:
just to confirm, this is an address on axelarnet? Otherwise we need a lot more information to route it back to another chain

Reply:
The requirement is quite vague. I don't think we are even sure how exactly will the refund be implemented, but according to @milapsheth this `refund_recipient` can be an ethereum address even when the event is emitted from the IBC path of the thing.

Reply:
do we need to go back to speccing then? A simple string is not enough information if the refund can go anywhere

Reply:
You can have EVM -> Cosmos -> EVM multihop where the refund needs to happen on the actual source, so the refund address is for that. You can also have Cosmos -> Cosmos -> Axelar -> EVM via packet forwarding where the refund address is a different cosmos chain.

Although, our multihop gas model hasn't been decided yet, so requirements are still vague. Since this is baked into consensus, I was thinking about keeping it as a string for flexibility, letting the relayer interpret it however it wants. There are some discussions going on about the multihop model rn, so if that concludes within a week or two, we can update this to be more specific.